.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
+
+enabled:
+  - align_double_arrow
+  - newline_after_open_tag
+  - ordered_use
+  - long_array_syntax
+  - php_unit_construct
+  - php_unit_strict
+
+disabled:
+  - psr0
```

Comment:
Yes I saw that. But it is not for the moment.

Does this will be broken if we leave this disabled fixer as is?